content/en/agent/basic_agent_usage/_index.md

```diff
@@ -256,13 +256,13 @@ Enabling JMX Checks forces the Agent to use more memory depending on the number
 {{< tabs >}}
 {{% tab "Agent v6 & v7" %}}
 
-* Agent Test version: 6.7.0
-* CPU: ~ 0.12% of the CPU used on average
-* Memory: ~ 60MB of RAM used (RSS memory)
-* Network bandwidth: ~ 86 B/s ▼ | 260 B/s ▲
+* Agent Test version: 7.34.0
+* CPU: ~ 0.08% of the CPU used on average
+* Memory: ~ 130MB of RAM used (RSS memory)
+* Network bandwidth: ~ 140 B/s ▼ | 800 B/s ▲
 * Disk:
-  * Linux 350MB to 400MB depending on the distribution
-  * Windows: 260MB
+  * Linux 850MB to 1.2GB depending on the distribution
```

Comment:
`depending on the distribution`: does this depend on the deb package vs. one of the rpm packages (RHEL or SUSE)? Or something else?

I'm just curious since the spread is much larger than in v6.7.0

Reply:
I was surprised too, quickly looking at the deb install we have 338MB just from new binaries:

```
87M	/opt/datadog-agent/embedded/bin/security-agent
92M	/opt/datadog-agent/embedded/bin/process-agent
159M	/opt/datadog-agent/embedded/bin/system-probe
```

Reply:
I checked on Amazon Linux 2 and Ubuntu and put the difference of disk usage before and after the installation.
If you think it's more relevant, I can use values from installers (which are between 700 and 800MB if I remember correctly)

Also, do you want me to check on SUSE and RedHat ?

Reply:
Hi! Is this open conversation a blocker to merging?

Reply:
Yes, let's wait for this conversation to be resolved before merging, thanks

I'm surprised there's such a large difference in the size after install of 7.34 between the deb and the rpm package.

@Kaderinho just checking, did your measurement method take into account the download of the package (deb/rpm) itself, or other caches that the package manager may keep after the install? (i.e. checking that you're not including the compressed package size, or other package-manager-related caches, in this measurement of the total size used on disk by the Agent after package installation)

Reply:
I did not pay attention to that so the measurements might include cached files related to the installation. I'll measure again more carefully.

Reply:
Checked again with cache removal and here are results I got : 

|              | Filesytem | /opt/datadog-agent |
|--------------|-----------|--------------------|
| Amazon Linux | 853MB     | 813MB              |
| Debian       | 860MB     | 832MB              |
| SUSE         | 837MB     | 813MB              |
| RHEL         | 876MB     | 813MB              |


Does the range 830-880MB seem good for you @olivielpeau ?

Reply:
Sounds great, thanks for looking into this 👍

Reply:
Done in [19ffb95](https://github.com/DataDog/documentation/pull/13311/commits/19ffb95be5a0b0e43666a2579f5c94d256fecc85)